### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

**Review by erh — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,8 +1,94 @@
...
+    "vision": "<your_vision_service_name>",
+    "WindowSeconds": <time_window_for_capture>,
+    "Classifications": {
+        "label": <confidence_score>,
+        "label": <confidence_score>
+    },
+    "Objects": {
+        "label": <confidence_score>,
+        "label": <confidence_score>
+    }
```
Comment (npentrel) [inline README.md:36]:
```suggestion
    "window_seconds": <time_window_for_capture>,
    "classifications": {
        "<label>": <confidence_score>,
        "<label>": <confidence_score>
    },
    "objects": {
        "<label>": <confidence_score>,
        "<label>": <confidence_score>
    }
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,94 @@
...
+| Name | Type | Inclusion | Description |
+| ---- | ------ | ------------ | ----------- |
+| `camera` | string | **Required** | The name of the camera to filter images for. |
+| `vision` | string | **Required** | The vision service used for image classifications or detections. |
+| `WindowSeconds` | float64 | Optional | The size of the time window during which images are buffered. When a condition is met, a confidence score for a detection/classification exceeds the required confidence score, the buffered images are stored, allowing us to see the photos taken in the N number of seconds preceding the condition being met.
+in seconds for capturing and sending images. |
+| `Classifications` | float64 | Optional | A map of
+classification labels and the confidence scores required for filtering. |
+| `Objects` | float64 | Optional | A map of object detection
+labels and the confidence scores required for filtering. |
```
Comment (npentrel) [inline README.md:56]:
```suggestion
| `window_seconds` | float64 | Optional | The size of the time window during which images are buffered. When a condition is met, a confidence score for a detection/classification exceeds the required confidence score, the buffered images are stored, allowing us to see the photos taken in the N number of seconds preceding the condition being met.
in seconds for capturing and sending images. |
| `classifications` | float64 | Optional | A map of
classification labels and the confidence scores required for filtering. |
| `objects` | float64 | Optional | A map of object detection
labels and the confidence scores required for filtering. |
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,94 @@
...
+
+### Example configurations:
+
+```json
+{
+    "camera": "my_camera",
+    "vision": "my_red_car_detector",
+    "WindowSeconds": 5,
+    "Objects": {
```
Comment (npentrel) [inline README.md:65]:
```suggestion
    "windowSeconds": 5,
    "objects": {
```

**Comment by npentrel** [comment]
@erh I think you have to merge

**Pull request merged**